benchmark/src/cmd_script.py

```diff
@@ -7,6 +7,18 @@
 
 import subprocess
 import argparse
+import os
+import time
+from grpc._channel import _InactiveRpcError
+from adlik_serving import PredictContext  # pylint: disable=import-error
+
+
+def _test_connect_of_serving(args):
```

Comment:
If return value has specific meaning, it should be clear. Normally, 'true' value always means 'success' or 'good' by subconscious thought.